rewards/veOcean-Data-Farming-Tutorial.md

```diff
@@ -76,5 +74,22 @@ All rewards are paid out in OCEAN tokens. On every Thursday the pink "Claim" but
 
 After the Lock End Date, then you can withdraw your principal OCEAN tokens on the [veOCEAN page](https://df.oceandao.org/veocean) on the left side panel.
 
+## Delegation
+
+Delegation allows you to temporarily transfer your veOCEAN allocation power to another wallet address. This feature enables you to earn a higher annual percentage yield (APY) by delegating to an address that efficiently manages your allocation, without the need for reallocation and transaction fees.
+
+If you have multiple wallets, delegation enables you to manage your allocations with just one wallet address.
+
+When you delegate, you transfer 100% of your veOCEAN allocation power for a limited period. After delegation, you cannot manage your allocations until the delegation expires. The delegation expiration date is the same as the veOCEAN lock end date at the time of delegation. If necessary, you can extend your lock before delegating. You can cancel your delegation at any time.
+
+Once delegated, rewards will be sent to the wallet address you delegated to. The delegation receiver is in charge of your rewards and the process of returning those back to you.
+
+To delegate, go to the DF Portal and select the wallet address you wish to delegate to. Click the Delegate button and sign the transaction. You can view information about your delegation in the My Delegations component. If needed, you can cancel delegation to regain your allocation power before the delegation expires.
+
+![](./images/veOCEAN-Delegation.png)
```

Comment:
Image should change base on @trizin latest comments.
Rewards will not be dispensed to the receiver.